apps/challenges/views.py

```diff
@@ -1210,6 +1210,14 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
                 }
                 send_slack_notification(message=message)
 
+            response = start_workers([challenge])
+            count, failures = response["count"], response["failures"]
+            logging.info("Total worker start count is {} and failures are: {}".format(count, failures))
+            if count:
+                logging.info("{} workers started successfully".format(count))
```

Comment:
@RishabhJain2018 @deshraj  need to know what to do in case of worker startup failure